### PR TITLE
variable,session: add search_path system variable.

### PR DESCRIPTION
**Pull request by Orion7r**
### What problem does this PR solve?

Add a PostgreSQL system variable named "search_path".

### What is changed and how it works?

What's Changed:

Add a system variable,named 'search_path',default value is 'public'.

How it Works:

Now，we can run the related SQL statements properly，like ' set search_path to public'

### CODE REVIEW

**Review by AmoebaProtozoa — APPROVED** [review]
LGTM

**Review by AmoebaProtozoa — APPROVED** [review]
LGTM

File: infoschema/tables.go [hunk 1]
```diff
@@ -59,6 +59,7 @@ import (
 	"github.com/pingcap/tidb/util/execdetails"
 	"github.com/pingcap/tidb/util/pdapi"
 )
+// todo Table names in system tables need to be replaced with lower case
```
Comment (pupillord) [inline infoschema/tables.go:62]: Here are two identical comments.

**Review by pupillord — CHANGES_REQUESTED** [review]

File: infoschema/tables.go (hunk 1 above)
Comment (Orion7r) [reply infoschema/tables.go]: Next commit fix it.

**Review by pupillord — APPROVED** [review]
LGTM

**Review by AmoebaProtozoa — APPROVED** [review]
LGTM

**Pull request merged**